src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,97 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { CommentToPost } from '../../types/comment-to-post';
+import { postComment } from '../../api/comments';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
-    >
-      Add a comment
-    </button>
-  </form>
-);
+interface Props {
+  postId: string,
+  reloadComments: () => void,
+}
+
+export const NewCommentForm: React.FC<Props>
+= ({ postId, reloadComments }) => {
+  const [data, setData] = useState<CommentToPost>({
+    postId: +postId,
+    name: '',
+    email: '',
+    body: '',
+  });
+
+  useEffect(() => {
+    const updatedData = { ...data };
+
+    updatedData.postId = +postId;
+    setData(updatedData);
+  }, [postId]);
```

Comment:
better to convert `postId` to number before passing to this component 1 time, instead of converting here in 3 places